src/navigation/index.js

```diff
@@ -1,27 +1,129 @@
-import * as React from 'react';
+import React, { useContext, useEffect, useState } from 'react';
+import { NativeModules } from 'react-native';
 import { NavigationContainer, DefaultTheme } from '@react-navigation/native';
 import { createBottomTabNavigator } from '@react-navigation/bottom-tabs';
 import Theme from '../theme/index';
 import { Login, Todos, Register } from '../screens';
+import Loading from '../components/utilities/loading';
+import { AppContext, useGlobalStateMgmt } from '../global-state';
+import { useToggle } from '../hooks/useToggle';
+import MaterialCommunityIcons from 'react-native-vector-icons/MaterialCommunityIcons';
 
+const { ForgeRockModule } = NativeModules;
 const Tab = createBottomTabNavigator();
 
 function Navigation() {
+  const [auth, setAuth] = useToggle();
+
+  useEffect(() => {
+    async function checkForToken() {
+      try {
+        if (!auth) {
+          await ForgeRockModule.frAuthStart();
+          const token = await ForgeRockModule.getAccessToken();
```

Comment:
I'm curious what this `getAccessToken` does. Does this attempt an Auth Code Flow when no tokens are stored locally?

Reply:
My thought here was if there is a token stored, i can set the context to true, or false if there is none.

Reply:
I have a question to George regarding this topic out. I don't think there is a current equivalent we have in the bridge to the TokenStorage.get